util/design/lib/OtpMemImg.py

```diff
@@ -329,12 +331,10 @@ def merge_item_data(self, part, item):
             item["value"] = common.check_bool(item["value"])
             mubi_val_str += "True" if item["value"] else "False"
             item["value"] = mubi_value_as_int(item["value"], item_width)
-        elif item["value"] in ["SOC_DBG_RAW", "SOC_DBG_PRE_PROD", "SOC_DBG_PROD"]:
+        elif item["name"] == "SOC_DBG_STATE":
```

Comment:
`item["value"]` didn't work correctly here, right?

Reply:
Correct. We use that downstream.